schema/sql/access.sql

```diff
@@ -0,0 +1,58 @@
+-- -*- mode: sql; sql-product: postgres -*-
+
+create user "api" noinherit;
+comment on role api is
+    $$API role used for API abstractions like PostgREST$$;
+alter role "api" with login;
+alter view v1."hosts" owner to api;
```

Comment:
This should exist next to the view itself; it's a workaround for a context sensitive operation.

Also, it would normally live near the policies that rely on it (though we don't have any policies right now)

Reply:
Externally facing users are created last, and then permissions assigned to them after that for ease of auditing external surface.

Reply:
The way to think about the anon user is that it's a "child" of the api user. Roots/Parents should be created before children.